internal/pkg/scaffold/ansible/go_mod.go

```diff
@@ -60,9 +60,12 @@ replace (
 	k8s.io/kubernetes => k8s.io/kubernetes v1.14.1
 )
 
-// Pinned to v2.10.0 (kubernetes-1.14.1) so https://proxy.golang.org can
-// resolve it correctly.
-replace github.com/prometheus/prometheus => github.com/prometheus/prometheus d20e84d0fb64aff2f62a977adc8cfb656da4e286
+replace (
+	sigs.k8s.io/controller-runtime => sigs.k8s.io/controller-runtime v0.2.0-rc.0
```

Comment:
Can we bump the `require` version and remove the `replace`? I think Go should pick up the correct `controller-runtime` version from the `github.com/operator-framework/operator-sdk` dependency. Ditto for the go and helm go.mod templates.

Maybe we need to change the `operator-sdk` require from `master` to the refactor branch until we merge back into master?

Reply:
The case I'm thinking of that a controller-runtime `replace` directive would solve is if the operator imports another package that has controller-runtime as a dependency at a higher version. Module mechanics would choose the higher version, which might break something in an operator.

Reply:
IIUC, in that case Go would then ignore the other package's dependency on a higher version of `controller-runtime`, which could also break something that that package is expecting from the higher version of CR.

So it's possible for things to break in either case, which will require user intervention either way. I'd lean toward respecting the default module versioning scheme and deferring to users on the use of `replace` where possible.

Reply:
Operator projects do have a direct dependency on controller-runtime (in main.go) and therefore pinning the version explicitly makes sense to me. However the `require` approach is fine with me because its unlikely that another project will use a version of controller-runtime that will break the operator, given Go's dependency resolution semantics.